core/controllers/payload_validator_test.py

```diff
@@ -109,13 +109,40 @@ def test_valid_args_do_not_raises_exception(self):
                 }
             }, {
                 'exploration_id': 'any_exp_id'
+            }),
+            ({
+                'apply_draft': 'true'
+            }, {
+                'apply_draft': {
+                    'schema': {
+                        'type': 'bool'
+                    }
+                }
+            }, {
+                'apply_draft': True
             })
         ]
         for handler_args, handler_args_schema, normalized_value_for_args in (
                 list_of_valid_args_with_schmea):
-            normalized_value_for_args, errors = payload_validator.validate(
-                handler_args, handler_args_schema, False)
+            normalized_value, errors = payload_validator.validate(
```

Comment:
This change is done because we are expecting to assert normalized_value (returned one) with normalized_value_for_args (expected one).
Initially both was same so technically they will always results True.